packages/core-design-system/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@harnessio/core-design-system",
+  "version": "0.0.1",
+  "description": "Harness Design System",
+  "type": "module",
+  "main": "src/core-style-esm.js",
+  "exports": {
+    "./core-styles": "./src/styles/core-imports.css",
+    "./oss-themes": "./src/styles/oss.css",
+    "./enterprise-themes": "./src/styles/enterprise.css",
+    "./styles-esm": "./src/core-styles-esm.ts"
```

Comment:
Ideally nothing should be exported from the `src` directory. All output should be built to a `dist` directory that is excluded from being committed.